### PR TITLE
element attribute commands

### PR DESCRIPTION
**Pull request by Jonahss**
8 more passing tests

### CODE REVIEW

File: lib/commands/element.js [hunk 1]
```diff
@@ -46,32 +46,19 @@ commands.getText = async function (elementId) {
 commands.clear = async function (elementId) {
   await this.setValue('', elementId);
 };
-
+*/
 commands.click = async function (elementId) {
```
Comment (moizjv) [inline lib/commands/element.js]: remove async

File: lib/commands/general.js [hunk 2]
```diff
@@ -46,6 +46,10 @@ commands.getScreenshot = async function () {
 
 */
 
+commands.back = async function () {
```
Comment (moizjv) [inline lib/commands/general.js]: no async

File: lib/commands/element.js [hunk 3]
```diff
@@ -26,52 +26,43 @@ commands.elementSelected = async function (elementId) {
...
-    value = keys.join("");
+    text = keys.join("");
   }
-  let el = this.getElement(elementId);
-  if (el.type !== "MockInputField") {
-    throw new errors.InvalidElementStateError();
-  }
-  el.setAttr('value', value);
+
+  let p = {
```
Comment (moizjv) [inline lib/commands/element.js]: p?

**Comment by jlipps** [comment]
I disagree that those `async`s should have been removed. In this case, while it's functionally equivalent not to have `async`, `async` serves as a kind of annotation that this is a function that returns a promise. Otherwise, I need to go and look at some _other_ function to know that I really need to `await` _this_ function if I want to use it correctly.

So my suggestion would be: if we have an async function, decorate it with `async`! It's a form of documentation as well.

**Pull request merged**